src/util/getDeriveAccount.ts

```diff
@@ -26,6 +26,9 @@ interface Data {
 }
 
 export default function getDeriveAccount({ ss58Format = 42, bridgeId, address }: Data): string {
+  if (!address) {
+    return address;
+  }
```

Comment:
This is a bit weird, why special case empty string?

Reply:
sorry , i'm not getting your question. Do you mean why i'm handling the empty address case?